src/components/IssueItem/__tests__/IssueItem.test.tsx

```diff
@@ -3,9 +3,8 @@ import { cleanup } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
```

Comment:
Are you testing the functionality of this page? you should be checking for example if SPDP-3 or Issue is inside the page

Reply:
Yes. I tested the IssueForm component by passing mocked `onSubmit` function to it

Reply:
It will be in the next task https://app.shortcut.com/deskpro/story/98884/youtrack-edit-issue